nix/sources.json

```diff
@@ -38,15 +38,15 @@
         "version": "962ecfed3a4fb656b5a91d89159291e00ed766bc"
     },
     "iohk-nix": {
-        "branch": "master",
+        "branch": "iquerejeta/include-blst",
```

Comment:
Is the `blst` not in `master` yet?

Reply:
Looks like PR has been merged: https://github.com/input-output-hk/iohk-nix/pull/528

We can go ahead and switch to `iohk-nix/master` now :1st_place_medal: